osu.Game/Screens/Edit/Timing/TapTimingControl.cs

```diff
@@ -183,12 +183,18 @@ private void load(OverlayColourProvider colourProvider, OsuColour colours)
 
         private void start()
         {
+            if (selectedGroup.Value == null)
+                return;
```

Comment:
Thoughts on adding these to `adjust{Bpm,Offset}` as well? Those cases don't look reachable from user interaction yet, but it's a bit weird to have null checked in only some branches.

Reply:
I wouldn't be against it.

Reply:
I'll go ahead and add them to the rest, then (with appropriate test coverage).